src/option_parser.c

```diff
@@ -187,20 +187,55 @@ int ini_get_bool(char *section, char *key, int def)
         }
 }
 
-char *clean_value(char *value)
+char *clean_value(char *value, int line_num)
```

Comment:
> This entire function is way too complicated at a whopping 6 levels of indentation(!!!) for something that can probably be done in a few lines in the `load_ini_file` function.

I agree, but if you look at other codebases, it's kind of par-the-course for quoting.

> What's more is that you assume that the first quote is always the start of the string, this means that something like String \"with quotes\" fails to parse

Oh! You are right.  TBH, I probably shouldn't have been pushing when I was so tired.  I'll simplify it a bit and fix that (by making it a bit more similar to the the original #126.